action.yml

```diff
@@ -59,19 +59,19 @@ runs:
       run: >
         ${{ github.action_path }}/vendor/bin/phpcs -s --standard=Drupal,DrupalPractice
         --extensions=${{ inputs.extensions }} ${{ inputs.path }} --ignore=${{ inputs.phpcs-ignore }}
-    - name: Copy-Pasta - Exclude
-      shell: bash
-      if: ${{ always() && inputs.phpcpd-exclude != '' }}
-      continue-on-error: ${{ inputs.phpcpd-continue == 'true' }}
-      run: >
-        ${{ github.action_path }}/vendor/bin/phpcpd --suffix=module --suffix=install --suffix=inc --suffix=theme --suffix=test --suffix=profile --suffix=${{ inputs.suffix }}
-        --exclude=${{ inputs.phpcpd-exclude }} ${{ inputs.path }}
-    - name: Copy-Pasta - Normal
+    - name: Copy-Paste
       shell: bash
-      if: ${{ always() && inputs.phpcpd-exclude == '' }}
+      if: always()
       continue-on-error: ${{ inputs.phpcpd-continue == 'true' }}
-      run: >
-        ${{ github.action_path }}/vendor/bin/phpcpd --suffix=module --suffix=install --suffix=inc --suffix=theme --suffix=test --suffix=profile --suffix=${{ inputs.suffix }} ${{ inputs.path }}
+      run: |
+        output=""
+        IFS=',' read -ra SUFFIX <<< ${{ inputs.suffix }}
+        for i in "${SUFFIX[@]}"; do
```

Comment:
I prefer not using `i` if it's not the index in a loop.

Reply:
Changed `i` to `suffix` to better indicate what is being used.